app/app.go

```diff
@@ -463,6 +481,52 @@ func NewApp(
 	return app
 }
 
+func inflationCalculator(ctx sdk.Context, minter minttypes.Minter, params minttypes.Params, bondedRatio sdk.Dec) sdk.Dec {
+	var deploymentParams deploymenttypes.Params
+	deploymentParamSubspace.GetParamSet(ctx, &deploymentParams)
+	tHalf := float64(deploymentParams.InflationDecayFactor)
+
+	// Number of hours in an year = 8766 (averaging the leap year hours)
+	// Number of minutes in an hour = 60
+	// Number of seconds in a minute = 60
+	// => 60 * 60 * 8766 = Number of seconds per year
+	t := ctx.BlockTime().Sub(genesisTime).Seconds() / (60 * 60 * 8766) // years passed
+	i := 100.0 * math.Pow(2, -t/tHalf)                                 // 100 = initial inflation (i0)
```

Comment:
Initial inflation was not 100%.  Let's make this a parameter as well.

Reply:
Done